articles/virtual-machines/windows/java.md

```diff
@@ -204,7 +162,6 @@ AvailabilitySet availabilitySet = azure.availabilitySets()
     .define("myAvailabilitySet")
     .withRegion(Region.US_EAST)
     .withExistingResourceGroup("myResourceGroup")
-    .withSku(AvailabilitySetSkuTypes.MANAGED)
```

Comment:
Use default SKU directly, there is no AvailabilitySetSkuTypes.MANAGED SkuType now.